src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,120 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, $catalog__card-width);
+  gap: 46px 48px;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    & {
+      @include square(2);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      @include square(3);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      @include square(4);
+    }
+  }
+
+  .card {
+    width: $catalog__card-width;
+    padding: 32px 16px 16px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    transition: transform 300ms;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+
+    &__image {
+      align-self: center;
+      width: 160px;
+      height: 134px;
+      margin: 0 0 40px;
+    }
+
+    &__title {
+      color: #060b35;
+      margin-bottom: 4px;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 500;
+
+      &:hover {
+        color: #34568b;
+      }
+    }
+
+    &__product-code {
+      margin-bottom: 16px;
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
+      color: #616070;
+    }
+
+    &__rating-field {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 24px;
+    }
+
+    &__rating-text {
+      color: #060b35;
+      font-size: 10px;
+      font-weight: 400;
+    }
+
+    &__price-field {
+      margin-bottom: 16px;
+      display: flex;
+      justify-content: space-between;
+    }
+
+    &__price-field-text {
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 400;
+      color: #616070;
+    }
+
+    &__price-field-number {
+      color: #060b35;
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+    }
+
+    &__buy-link {
+      box-sizing: border-box;
+      display: block;
+      text-align: center;
+      width: 100%;
+      height: 40px;
+      background-color: $page-design-color;
+      border: 1px solid $page-design-color;
+      border-radius: 5px;
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 38px;
+      text-decoration: none;
+      color: #fff;
+
+      &:hover {
+        background-color: #fff;
+        color: $page-design-color;
```

Comment:
The variable `$page-design-color` is used here, but ensure that it is defined elsewhere in your SCSS files. If not, this will cause a compilation error.